CMakeLists.txt

```diff
@@ -1,6 +1,14 @@
 cmake_minimum_required (VERSION 2.8.11)
 project (benchmark)
 
+foreach(p
```

Comment:
ah, that's a shame. does setting the policy to 'new' mean we have different behaviour and we have bugs in the existing cmake config, i wonder.